CHANGELOG.next.asciidoc

```diff
@@ -112,6 +112,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 *Heartbeat*
 
 - Made monitors.d configuration part of the default config. {pull}9004[9004]
+- Fixed rare issue where TLS connections to endpoints with x509 certificates missing either notBefore or notAfter would cause the check to fail with a stacktrace.  {pull}9566[9566]
```

Comment:
Does not seem to belong to this PR?

Reply:
Technically no, but I moved it since it was in the wrong spot. Should I make a new PR?

Reply:
Just realised now that you moved it only and was not added. All good.